apps/web/contexts/common-contexts.ts

```diff
@@ -4,6 +4,11 @@ import {createClient} from '@supabase/supabase-js'
 export const supabase = createClient(
   'https://hhnxsazpojeczkeeifli.supabase.co',
   'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImhobnhzYXpwb2plY3prZWVpZmxpIiwicm9sZSI6ImFub24iLCJpYXQiOjE2NjAyNjgwOTIsImV4cCI6MTk3NTg0NDA5Mn0.ZDmf1sjsr-UxW2bPgdj3uaqJNUSqkZh8vCB1phn3qqs',
+  {
+    auth: {
+      autoRefreshToken: true,
```

Comment:
Note: we should default into auto refreshing these (and still need to sanity check people get logged out when it finally expires)

Reply:
surprised it's not the supabase default already.